app/lib/shared/utils.dart

```diff
@@ -413,3 +413,29 @@ String createUuid([List<int> bytes]) {
     bytes.sublist(10).map(formatByte).join(),
   ].join('-');
 }
+
+extension DateTimeExt on DateTime {
+  /// `true` if `relativeTo] - this > duration`, where [relativeTo] defaults
+  /// to [DateTime.now()].
+  ///
+  /// In other words, if this [DateTime] is more than [duration] time ago, since
+  /// [relativeTo] (or now, if not given). This will return true.
+  ///
+  /// **Example**
+  /// ```
+  /// final start = DateTime.now();
+  ///
+  /// // do some work...
+  ///
+  /// if (start.isTimeAgo(Duration(minutes: 15))) {
+  ///   // Start is more than 15 minutes ago.
+  /// }
+  /// ```
+  ///
+  bool isTimeAgo(Duration duration, [DateTime relativeTo]) {
```

Comment:
I have never felt the need for such utility, as it seems to be about the same length-wise compared to the direct diff() + comparison. Are you planning more like that / into this method?

Reply:
I felt like I needed it because I'm always doing this wrong... remember last time with the email logic.